Tools/px_uploader.Dockerfile

```diff
@@ -19,12 +21,20 @@ FROM python:alpine3.14
 
 WORKDIR /firmware
 
-ENTRYPOINT ["/bin/px_uploader.py"]
+ENTRYPOINT ["/entrypoint.sh"]
 
 # dependency of px_uploader.py
 RUN pip3 install --user pyserial
 
 ADD px4-firmware/Tools/px_uploader.py /bin/
+ADD px4-firmware/Tools/px_uploader.entrypoint /entrypoint.sh
 
 # copy /bin/* -> /firmware/*
 ADD bin/ /firmware/
+
+ADD px4-firmware/ssrc_config/indoor_lighthouse/config.txt /flight_modes/indoor_config.txt
+ADD px4-firmware/ssrc_config/HIL/config.txt /flight_modes/hitl_config.txt
+ADD px4-firmware/ssrc_config/HIL/ethernet/config.txt /flight_modes/hitl_eth_config.txt
```

Comment:
Not strictly related to this issue, but I got an idea

This translation layer adds complexity: "HIL/config.txt is known as hitl_config.txt in context of px4-post-update-init"

=> how important is it that `ssrc_config/HIL/config.txt` in this repo is at that location, and not already as `ssrc_config/hitl_config.txt` ?

If that was the case, we could simply have `ADD px4-firmware/ssrc_config/ /flight_modes/`

Benefits:

- No translation layer of filenames being different in context of px4-post-update-init
- No need to make changes to this line each time we add/rename/remove config files

Also another idea: I realize that outdoor config is the default case and thus doesn't need a config file because its config file would effectively be empty. BUT, what if we added `outdoor_config.txt` with almost empty content:

```
# Outdoor is the default case that firmware is pre-configured for, thus no parameters required.
```

This would mean we don't have to have separate handling for "outdoor doesn't use a file" here: https://github.com/tiiuae/px4-post-update-init/blob/ff4c6cf09607b7c1d9891546d535c536c0afbf68/main.go#L120

And we could simply have the logic be to read `/flight_modes/config_<flight_mode>.txt` and not need these branches:

```go
	switch mode {
	case "outdoor":
		log.Println("# Outdoor flight mode, no config files uploaded")
		return success
	case "indoor":
		log.Println("# Indoor flight mode")
		config_path += "config_indoor.txt"
	case "hitl":
		log.Println("# HITL mode")
		config_path += "config_hitl.txt"
	}
```

(note also: above is one more layer where we currently need to make modifications if we introduce new config files, for example I notice `px4-post-update-init` is missing a handler for `hitl_eth`, and now I'm wondering if that's intentional?)

Reply:
Hmm it seems:
- this `Dockerfile` adds `indoor_config.txt`
- looking at the Go code, it refers to `config_indoor.txt`

One of those seems incorrect?

Reply:
There is typo in Dockerfile, I meant it to be config_<mode>.txt

Reply:
This common code for config file handling sounds good and it sure make it easier to add or modify flight modes.